tests/redisearch.ts

```diff
@@ -28,8 +28,19 @@ describe('RediSearch Module testing', async function() {
         });
         await client.connect();
         await redis.connect();
+        try {
+            await client.redis.flushall();
+            await client.dropindex(index);
+            await client.dropindex(`${index}1`);    
+            await client.dropindex(`${index}-searchtest`);    
+            await client.dropindex(`${index}-droptest`);    
+        } catch (error) {
+            // console.error(error);
+        }
     })
```

Comment:
Why the try and catch? does one of this return an error?

Reply:
if indexes do not exist they throw an error.

Reply:
Each run is done clean by the CI, that's why there is no data cleanup or setup. At the same time, If you would restart your redis server each time, the data will be clean. therefore, all of the cleanup you did here is pretty redundant.
It is correct to test locally, but I did not implement it yet and at this moment the tests for the Redis server the CI sets up via docker. At the same time, you could use the modules docker images to set it up easily via docker instead of adding any additional files.